identifiers/country-es/provincia.csv

```diff
@@ -26,7 +26,14 @@ ocd-division/country:es/autonomous_community:cm/province:cu,Cuenca
 ocd-division/country:es/autonomous_community:cm/province:gu,Guadalajara
 ocd-division/country:es/autonomous_community:cm/province:to,Toledo
 ocd-division/country:es/autonomous_community:cn/province:gc,Las Palmas
+ocd-division/country:es/autonomous_community:cn/province:gc-1,Gran Canaria
```

Comment:
My understanding is these additional OCD IDs aren't actually provinces, but instead districts used for elections right?

I think `ed` would be a more appropriate type to mark them as electoral districts and nest it at whatever level of government the decision of the creation of these electoral districts is being made. For example, if at the province-level they are deciding to subdivide into these electoral districts then the current nesting looks fine.

I also think it'd be better to have these in a separate file since they aren't provinces.